modules/nf-core/smncopynumbercaller/main.nf

```diff
@@ -0,0 +1,49 @@
+#!/usr/bin/env nextflow
+nextflow.enable.dsl = 2
+def SMNCOPYNUMBERCALLER_VERSION = 'SMNCopyNumberCaller commit 3e67e3b on Feb 8, 2020'
+// No versioning included with this program; added github commit from:
+// https://github.com/Illumina/SMNCopyNumberCaller
+
+process SMNCOPYNUMBERCALLER {
+    tag "$meta.id"
+    label 'process_low'
+
+    if (params.enable_conda) {
+        exit 1, "Conda environments cannot be used with SMNCopyNumberCaller at the moment. Please use Docker or Singularity containers."
+    }
+    container "clinicalgenomics/smncopynumbercaller:v1.1.2"
```

Comment:
Is there a reason that this tool can't be packaged on BioConda? See [this bytesize talk](https://nf-co.re/events/2022/bytesize-40-software-packaging) if you're new to the process..

Reply:
Thanks, I will package it in bioconda.